__tests__/api.test.js

```diff
@@ -96,6 +99,65 @@ describe('GET requests', () => {
             expect(commentZero).toHaveProperty('created_at');
         })
     })
+
+
+    describe.only('POST REQUESTS', () => {
+        test('200: responds with posted article (ticket 7)', () =>{
+            return request (app)
+            .post('/api/articles/2/comments')
+            .expect(201)
+            .send({'username': 'butter_bridge', 'comment':'A very interesting opinion'})
+            .then(({body : {post}}) => {
+                expect(post).toHaveProperty('comment_id', 19)
+                expect(post).toHaveProperty('body', 'A very interesting opinion')
+                expect(post).toHaveProperty('article_id', 2)
+                expect(post).toHaveProperty('author', 'butter_bridge')
+            })
+
+        })
+        test('400: responds with an error when username does not exist', () => {
+            return request (app)
+            .post('/api/articles/2/comments')
+            .expect(400)
+            .send({
+                'username' : "notReal",
+                'comment' : ' Whocares',
+            })
+            .then(({body}) => {
+                expect(body).toHaveProperty('msg')
```

Comment:
as well as checking it has the property I'd check that it contains the message you're expecting